content/the-ember-times-issue-182.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: The Ember Times - Issue No. 182
 authors:
-  - the-crowd # replace with real authors from the author folder (add yourself if you're not there)
+  - Tim Foster # replace with real authors from the author folder (add yourself if you're not there)
```

Comment:
Sorry, the directions aren't too clear. Here, you'll want to add the identifier corresponding to your name.

```suggestion
  - tim-foster # replace with real authors from the author folder (add yourself if you're not there)
```